.github/workflows/main.yml

```diff
@@ -1,14 +1,19 @@
----
 name: Build
 on:
   push:
     branches:
       - master
     tags:
       - v*
+    paths-ignore:
+      -'fastlane/**'
+      -'README.md'
   pull_request:
     branches:
       - master
+    paths-ignore:
+      -'fastlane/**'
+      -'README.md'

```

Comment:
If we want to ignore this paths for pull request, why not.

But I would keep them for the master branch.